wolfssl/wolfcrypt/sha.h

```diff
@@ -67,24 +75,32 @@ typedef struct Sha {
         word32  loLen;     /* length in bytes   */
         word32  hiLen;     /* length in bytes   */
         word32  buffer[SHA_BLOCK_SIZE  / sizeof(word32)];
-        #ifndef WOLFSSL_PIC32MZ_HASH
-            word32  digest[SHA_DIGEST_SIZE / sizeof(word32)];
-        #else
-            word32  digest[PIC32_HASH_SIZE / sizeof(word32)];
```

Comment:
I had reworked the SHA_DIGEST_SIZE to use PIC32_HASH_SIZE, but that is not a good cleanup, so revert to original.